cg/constants.py

```diff
@@ -62,6 +62,7 @@
 
 # tags for storing analyses in Housekeeper
 TAGS = {
```

Comment:
```suggestion
HK_TAGS = {
```
I think this is more mnemonic. So that ones does not need to read the comment above to understand the context. Particurlarly, when this is imported in another module.